tool_path_planner/src/plane_slicer_raster_generator.cpp

```diff
@@ -514,7 +513,7 @@ boost::optional<ToolPaths> PlaneSlicerRasterGenerator::generate()
   double min_coeff = dist.minCoeff();
 
   // Calculate the number of planes to cover the bounding box along the direction vector
-  auto num_planes = static_cast<std::size_t>(std::ceil((max_coeff - min_coeff) / config_.raster_spacing));
+  std::size_t num_planes = static_cast<std::size_t>(std::ceil((max_coeff - min_coeff) / config_.raster_spacing));
```

Comment:
Recommend keeping the auto here. See [here](https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-auto.html?highlight=static_cast#cast-expressions)